tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/sequencefile/SequenceFileScanner.java

```diff
@@ -105,6 +105,11 @@ public void init() throws IOException {
     }
 
     String delim  = meta.getProperty(StorageConstants.SEQUENCEFILE_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    if (delim == null || delim.isEmpty()) {
+      delim = meta.getProperty(StorageConstants.TEXT_DELIMITER, StorageConstants.DEFAULT_FIELD_DELIMITER);
+    }
+    meta.getPropertySet().set(StorageConstants.TEXT_DELIMITER, delim);
```

Comment:
If users only set ``SEQUENCEFILE_DELIMITER``, ``TextLineSerDe`` can't delimit records because ``TextLineSerDe`` use ``TEXT_DELIMITER`` as the delimiter. I modified codes again as follows. The value of ``SEQUENCEFILE_DELIMITER`` will only use if ``TEXT_DELIMITER`` doesn't exit and ``SEQUENCEFILE_DELIMITER`` exists only.